src/components/PostIndex.jsx

```diff
@@ -22,6 +28,15 @@ const PostIndex = ({ post }) => {
         <Text testID={`post-description-${post.id}`} style={styles.description}>
           {post.description}
         </Text>
+        <Octicons
+          name="flame"
+          size={24}
+          color="black"
+          testID={`likeButton-${post.id}`}
+          onPress={() => { LikeService.create(post.id, credentials); PostService.index() }}
+        >
+          <Text testID={`likeCount-${post.id}`}>{post.likes}</Text>
+        </Octicons>
```

Comment:
it's very likely a moot point since the plan is to use websockets, but I think this re-rendering of the index is a bit inefficient

Reply:
It doesn't re-render like counts without `PostService.index()`